developers/ide/vscode.md

```diff
@@ -15,7 +15,13 @@ The following steps will only need to be done once to setup both VSCode and your
    
 2. Clone the addons (https://github.com/openhab/openhab-addons.git or preferably your own fork) to %BASE%\openhab-addons
```

Comment:
```suggestion
1. Clone the addons (https://github.com/openhab/openhab-addons.git or preferably your own fork) to %BASE%\openhab-addons
```

Reply:
Why do you want to change all numbers to 1.?

Reply:
See explanation in the other change request. 😃